learning_resources_search/serializers.py

```diff
@@ -293,7 +270,11 @@ def serialize_bulk_courses(ids):
     Args:
         ids(list of int): List of course id's
     """
-    for learning_resource in LearningResource.objects.filter(id__in=ids):
+    for learning_resource in (
+        LearningResource.objects.select_related(*LearningResource.related_selects)
+        .prefetch_related(*LearningResource.prefetches)
+        .filter(id__in=ids)
```

Comment:
Hoping that `select_related` and `prefetch_related` here might make the queries more efficient.